pkg/cmd/root.go

```diff
@@ -0,0 +1,58 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/redhat-developer/henge/pkg/types"
+)
+
+const cliLong = `
+Henge takes a docker-compose.yml file and converts it into a openshift or kubernetes artifacts,
```

Comment:
Yes we can https://github.com/redhat-developer/henge/pull/32/files#diff-a7f30f84a9d7d21cfedcc79a96ac9037R31